src/index.html

```diff
@@ -11,6 +11,31 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="link"
+      method="get"
+      class="formBig"
+      data-qa="big"
+    >
+      <input
+        type="text"
+        class="searchBig"
+        name="firstBox"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+      >
+    </form>
+
+    <form action="link"
+      method="get"
+      class="formSmall"
```

Comment:
Here is how to make block with modifier
So you will be not repeated with styles. Just add those style to form-small that have difference
```suggestion
      class="form form--small"
```